policy/modules/system/sysnetwork.te

```diff
@@ -363,6 +363,7 @@ term_dontaudit_use_all_ttys(ifconfig_t)
 term_dontaudit_use_all_ptys(ifconfig_t)
 term_dontaudit_use_ptmx(ifconfig_t)
 term_dontaudit_use_generic_ptys(ifconfig_t)
+term_use_unallocated_ttys(ifconfig_t)
```

Comment:
What scenario is this for?

Reply:
If we can get more info that could be helpful. It's not particularly concerning but understanding it could be helpful since sounds like a corner case.